src/main/java/edu/ucsb/cs156/spring/backenddemo/services/JokeQueryService.java

```diff
@@ -1,23 +1,48 @@
 package edu.ucsb.cs156.spring.backenddemo.services;
 
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.List;
+import java.util.Map;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
 import org.springframework.boot.web.client.RestTemplateBuilder;
 import org.springframework.stereotype.Service;
 import org.springframework.web.client.HttpClientErrorException;
 import org.springframework.web.client.RestTemplate;
 
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+
 
+@Slf4j
 @Service
 public class JokeQueryService {
+    ObjectMapper mapper = new ObjectMapper();
```

Comment:
🤓  is this unnecessary? I realized that it's not used anywhere after this line. Maybe the import statement can be removed too